src/server.ts

```diff
@@ -44,6 +45,7 @@ const build = (opts: FastifyServerOptions): FastifyInstance => {
   });
 
   Auth.assertEnvironmentSetup();
+  checkRedisConnection();
```

Comment:
Feels like this should await somehow? Is the idea that it will crash the app whenever it completes async? Does it work?

Reply:
Yes exactly, the idea is for it to crash whenever it completes and it works.